docusaurus.config.ts

```diff
@@ -94,11 +94,11 @@ const config: Config = {
           position: 'left',
           items: [
             {
-              to: '/docs/next/standards/overview/',
+              to: '/docs/standards/overview/',
               label: 'Standards',
             },
             {
-              to: '/docs/next/non-functional/overview/',
+              to: '/docs/non-functional/overview/',
```

Comment:
@werner-roman these changes were made because I think it's better to link to current (In our case Jupiter) instead of next?